common/djangoapps/student/views.py

```diff
@@ -874,20 +849,6 @@ def change_enrollment(request, check_access=True):
 
         # Otherwise, there is only one mode available (the default)
         return HttpResponse()
-
-    elif action == "add_to_cart":
```

Comment:
@rlucioni Yes, since the new auth completion view uses the [enrollment API](https://github.com/edx/edx-platform/tree/master/common/djangoapps/enrollment), since no other edx-platform code seemed to use this functionality here now that I did that refactor, and since the enrollment API is available for external enrollment calls via API, it seems there is no need to leave this code here.

Can you give me a link to an example on staging or instructions for reproducing the issue?